homeassistant/components/climate/zwave.py

```diff
@@ -158,6 +158,16 @@ def update_properties(self):
                     continue
                 if self._zxt_120:
                     continue
+                self._target_temperature = int(value.data)
+                _LOGGER.debug("Setpoint get SET_TEMP_TO_INDEX=%s and"
+                              " self._current_operation=%s",
+                              SET_TEMP_TO_INDEX.get(self._current_operation),
+                              self._current_operation)
+                break
+            _LOGGER.debug("Setpoint get not matching SET_TEMP_TO_INDEX=%s and"
```

Comment:
Think you can clarify this debug line? Not sure what exactly `setpoint get not` means.